modules/oe_list_pages_open_vocabularies/tests/src/FunctionalJavascript/OpenVocabulariesFiltersTest.php

```diff
@@ -0,0 +1,388 @@
+    $values = [
+      'title' => 'one yellow fruit',
+      'type' => 'content_type_one',
+      'body' => 'this is a banana',
+      'status' => NodeInterface::PUBLISHED,
+      'field_select_one' => 'test1',
+      'field_open_vocabularies' => [
+        'target_id' => $term_1->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+    $values = [
+      'title' => 'another yellow fruit',
+      'type' => 'content_type_one',
+      'body' => 'this is a lemon',
+      'status' => NodeInterface::PUBLISHED,
+      'field_select_one' => 'test2',
+      'field_open_vocabularies' => [
+        'target_id' => $term_2->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->modify('+ 5 days')->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    // Create some test nodes to index and search in.
+    $date = new DrupalDateTime('09-02-2021');
+    $values = [
+      'title' => 'Sun',
+      'type' => 'content_type_two',
+      'body' => 'This is the sun',
+      'status' => NodeInterface::PUBLISHED,
+      'field_open_vocabularies' => [
+        'target_id' => $term_1->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    $values = [
+      'title' => 'Grass',
+      'type' => 'content_type_two',
+      'body' => 'This is grass',
+      'status' => NodeInterface::PUBLISHED,
+      'field_open_vocabularies' => [
+        'target_id' => $term_2->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->modify('+ 5 days')->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    /** @var \Drupal\search_api\Entity\Index $index */
+    $index = Index::load('node');
+    $index->indexItems();
+  }
+
+  /**
+   * Test exposed filters and default filters configuration.
+   */
+  public function testListPagePluginFiltersFormConfiguration(): void {
+    $admin = $this->createUser([], NULL, TRUE);
+    $this->drupalLogin($admin);
+    $this->goToListPageConfiguration();
+    $actual_entity_types = $this->getSelectOptions('Source entity type');
+    $expected_entity_types = [
+      '' => '- Select -',
+      'node' => 'Content',
+      'taxonomy_term' => 'Taxonomy term',
+    ];
+    $this->assertEquals($expected_entity_types, $actual_entity_types);
+    // By default, Node is selected if there are no stored values.
+    $this->assertOptionSelected('Source entity type', 'Content');
+
+    $actual_bundles = $this->getSelectOptions('Source bundle');
+    $expected_bundles = [
+      'content_type_one' => 'Content type one',
+      'content_type_two' => 'Content type two',
+      '' => '- Select -',
+    ];
+    $this->assertEquals($expected_bundles, $actual_bundles);
```

Comment:
This and all the previous assertions are duplicated from other tests that suit them better I think, we could take them out of this particular test class and save some precious precious time.